.github/workflows/test-k8s-service-templates.yml

```diff
@@ -153,77 +146,26 @@ jobs:
             exit 1
           fi
 
-      - name: Prepare inputs for the diff
-        if: always()
-        run: |
-          set -Eeuo pipefail
-          
-          process() {
```

Comment:
V CI mame nejake scripts, ktore generuju PR comment:
![image](https://user-images.githubusercontent.com/19371734/201310272-e4fe7292-3591-42cb-8305-e812d17b62ce.png)

Aby sa to dalo pouzit aj pre Buffer API, tak som skripty presunul do `provisioning/common/scripts`.

Myslim, ze sa to nemusi kontrolovat podrobne, ze tu funguje je vidiet podla toho, ze comment sa vygeneroval.